aws/resource_aws_iam_group_membership.go

```diff
@@ -35,6 +36,18 @@ func resourceAwsIamGroupMembership() *schema.Resource {
 				Required: true,
 				ForceNew: true,
 			},
+
+			"remove_unmanaged_members": &schema.Schema{
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  false,
```

Comment:
This is a breaking change for the resource as written. It should default to `true` to keep backwards compatibility.